cpp/oneapi/dal/algo/jaccard/detail/select_kernel.hpp

```diff
@@ -23,46 +23,43 @@
 
 namespace oneapi::dal::preview::jaccard::detail {
 
-template <typename Policy, typename Topology>
-struct ONEDAL_EXPORT backend_base {
-    virtual vertex_similarity_result operator()(const Policy &ctx,
-                                                const descriptor_base &descriptor,
-                                                const Topology &data,
-                                                void *result_ptr) = 0;
-    virtual ~backend_base() {}
-};
+template <typename Policy, typename Descriptor, typename Topology>
```

Comment:
ONEDAL_EXPORTs removed here, is it right?

Reply:
It is fully in header files - that is right removal.